rfcs/045-sierra-work-relationships/README.md

```diff
@@ -0,0 +1,377 @@
+here is a distillation of our current understanding of this issue, and the 
+practicalities of implementation.
+
+RFC-044 is a year old, and engendered a large amount of discussion, making 
+it unwieldy to review. Much of the information in that document is still 
+valid, and provides essential context, so it is left intact.
+
+These RFCs concern the linking of Works to Series, and of Works to other Works
+coming from Sierra.
+
+It is apparent that these are two different things, with two different 
+manifestations in the API and Website. Though they do have some common
+features.
+
+## Summary
+
+At the broadest level, there are three features to be added resulting from this RFC.
+
+* Linking of Works to Series
+* Linking of parent Works to child Works
+* Linking of child Works to parent Works
+
+## API change
+
+This requires a change to the API:
+* `id` is currently required in partOf objects, it needs to be optional.
+* a new field needs to be added to part and partOf objects: `subPart`
+
+## Preliminary Work
+
+We must ensure that the website will not attempt to render inappropriate
+hierarchies when we start adding new part/partOf relationships.
+
+## Staged development proposal
+
+1. All parent relationships described in this document to be displayed as series links. 
+   - This includes 773 relationships with identifiers, which will eventually link between Works.
+   - The subpart ($g or $v) is ignored
+2. Improve handling of identified Work->Work links for hierarchical display.
+3. Improve handling of asymmetric partOf links to link directly by id.
+4. Display the subPart if desired.
+
+This approach allows us to first deliver an improvement to the linking of
+as many records as possible, and then iterates over _how_ that linking
+can be improved for those records where we have more information.
+
+It also represents a cone of uncertainty.  It is clear that we want to link
+from Works to Series via a filtered search for the Series title.  The 
+correct presentation of links between parent and child works is less clear. 
+In particular, how best to handle asymmetric links.
+The value of presenting the subPart text to users is not clear.
+
+## Relevant MARC fields
+
+* [440 - Series Statement/Added Entry-Title](https://www.loc.gov/marc/bibliographic/bd440.html)
+* [490 - Series Statement](https://www.loc.gov/marc/bibliographic/bd490.html)
+* [773 - Host Item Entry](https://www.loc.gov/marc/bibliographic/bd773.html)
+* [774 - Constituent Unit Entry](https://www.loc.gov/marc/bibliographic/bd774.html)
+* [830 - Series Added Entry-Uniform Title](https://www.loc.gov/marc/bibliographic/bd830.html)
+
+
+## Common features
+
+### Most Precise Wins
+
+The 773 field is not restrictive about what it can point to, whereas the 4x0 and 830 
+fields explicitly refer to a Series.  Records with a 773 field that matches
+one of the series fields should ignore the 773 field and just present it
+as a Series link.
```

Comment:
Yes. Normally they are in a 773/774 pairing.  In that case it should become a Work->Work relationship.

This section is about the "winning" concept where they do match.  It may be a bit redundant, because the likelihood is that a 773 matching a Series field will not have a $w, so it would be treated as a Series anyway.

It is possible that there is no 4x0 or 830 _or even_ corresponding 774.  That is also expected to behave like a series. This is described below see the "James Gardiner Collection" example.

Despite the possible redundancy, I wanted to include this section to ensure that the situation below has been considered.